src/main/kotlin/no/nav/syfo/behandlerdialog/api/domain/BehandlerdialogDTO.kt

```diff
@@ -0,0 +1,9 @@
+package no.nav.syfo.behandlerdialog.api.domain
+
+import java.util.*
+
+data class BehandlerdialogDTO(
+    val personIdent: String,
+    val behandlerRef: UUID,
+    val tekst: String,
+)
```

Comment:
Flere ting her:
1. Trenger vi personident i DTOen, hvis dette uansett sendes i header?
2. Er behandlerRef en UUID, eller personident til behandler?
3. Er `tekst` et bra navn? `melding`, `meldingstekst`, `innhold`, `dialoginnhold` er alterantiver

Reply:
1. Tror ikke vi trenger personident her.
2. behandlerRef er en UUID (pleier å bruke String for det i DTOene, men funker vel med UUID også). Vi får denne fra behandler-apiet til isdialogmelding.
3 og 4. `tekst` er greit det, gå jo fint an å endre senere om det bli nødvendig hvis får flere felter f.eks.. Tenker i så fall at denne DTOen bør hete noe med melding (det er en DTO for å opprette en melding til behandler).